toolbox/fdc3-for-web/fdc3-web-impl/test/features/heartbeat.feature

```diff
@@ -32,6 +32,10 @@ Feature: Heartbeat Messages Between Apps and Server
       | heartbeatEvent   | a1            | libraryApp |
       | heartbeatEvent   | a1            | libraryApp |
     And I shutdown the server
+    And I get the heartbeat times
```

Comment:
note we switched away from the the heartbeat time to quoting the `eventUuid` of the heartbeat as `payload.heartbeatEventUuid`